server/odc-service/src/main/java/com/oceanbase/odc/service/db/schema/syncer/object/DBExternalTableSyncer.java

```diff
@@ -58,6 +67,19 @@ protected void preDelete(@NonNull Set<Long> toBeDeletedIds) {
         userPermissionRepository.deleteByPermissionIds(permissionIds);
     }
 
+    @Override
+    public boolean supports(@NonNull DialectType dialectType, @NonNull Connection connection) {
+        boolean externalTableSupported;
+        try {
+            externalTableSupported = tableService.checkExternalTableSupported(dialectType, connection);
+        } catch (SQLException e) {
+            log.warn("checkSupportObtainExternalTableList failed, dialectType:{}, connection:{}", dialectType,
```

Comment:
don't print the connection, there are no more info can be printing,
```java
log.warn("Failed to check if external table supported", e);
```

this log printing will be okay

Reply:
ok